orderer/consensus/smartbft/sync_buffer.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package smartbft
+
+import (
+	"sync"
+
+	"github.com/hyperledger/fabric-protos-go/common"
+	"github.com/pkg/errors"
+)
+
+type SyncBuffer struct {
+	blockCh  chan *common.Block
+	stopCh   chan struct{}
+	stopOnce sync.Once
+}
+
+func NewSyncBuffer() *SyncBuffer {
+	return &SyncBuffer{
+		blockCh: make(chan *common.Block, 10),
```

Comment:
can we increase this buffer to 100 by default? and ideally this will be configurable. can we open an issue to make this configurable?

Reply:
sure
https://github.com/hyperledger/fabric/issues/4715